dockerfiles/test/Dockerfile

```diff
@@ -1,11 +1,29 @@
-FROM mhart/alpine-node:latest
-RUN apk update && apk upgrade && \
-    apk add --no-cache \
-    udev \
-    xvfb \
-    chromium \
-    ca-certificates \ 
-    mongodb \
-    git
-RUN mkdir -p /data/db # create default mongodb data directory
-RUN apk add --no-cache bash
\ No newline at end of file
+# by :13.0.1 at least, alpine-node started including comming with old versions of chromium and mongodb installed
+# No luck upgrading or uninstalling them, and they were breaking testcafe and cleanup_dev_dbs.sh respectively
+# 12.9.0 was the previous image I built from/knew to be good, not necessarily the latest pre-funny buisness
+# ... can't stay locked to this forever though
```

Comment:
13.0.1 is the just-released version of alpine-node, right? so this might be a temporary problem, or at least not something we need to fix immediately.

Reply:
Yeah, 12.9.0 is from late August this year. Fingers crossed future alpine-node drops, or at least upgrades, its mongodb/chromium dependencies. We shouldn't get too out of date though, becomes a security hole if we aren't updating our images (limited somewhat because these are just for CI, not prod).

I might dig in to WHY it requires them some time, as doesn't seem like they should be in there, could be something they overlooked in the latest releases.

Reply:
Especially since they've self-branded as "minimal"

Reply:
Exactly, chromium and mongodb are practically maximal